src/index.ts

```diff
@@ -143,3 +147,53 @@ export function submitEmail(fn: (target, widget, email) => any): void {
 }
 
 if (window) asyncLoad();
+
+if (window && !window.SaaSquatchDoNotAutoDrop) {
```

Comment:
More isometric was of checking this

```suggestion
if (typeof document !== "undefined" && !window.SaaSquatchDoNotAutoDrop) {
```